src/features/books/views/AddBook.tsx

```diff
@@ -6,13 +6,15 @@ import DataFetchingError from "@/components/DataFetchingError/DataFetchingError"
 import { useEffect } from "react"
 import { getAllGenres } from "@/features/genres/genresThunk"
 import Loader from "@/components/Loader/Loader"
+import { addBook } from "../booksThunk"
 
 const AddBook = () => {
   const { t } = useTranslation(["books"])
   const dispatch = useAppDispatch()
   const { isGenresLoading, isGenresError, genres } = useAppSelector(
     (state) => state.genres,
   )
+
   useEffect(() => {
     if (!genres) {
```

Comment:
imagine a situation where more than two users are being logged in and one of them adds a new genre. you should be able to fetch the latest data from the database